docs/docs/graphql.md

```diff
@@ -1253,6 +1253,433 @@ export const getWeather = async ({ input }: WeatherInput) {
+"""
+input CreatePostInput {
+  "The content of a post."
+  body: String!
+
+  "The title of a post."
+  title: String!
+}
+
+scalar Date
+
+scalar DateTime
+
+scalar JSON
+
+scalar JSONObject
+
+"""
+About mutations
+"""
+type Mutation {
+  "Creates a new Contact."
+  createContact(input: CreateContactInput!): Contact!
+
+  "Creates a new Post."
+  createPost(input: CreatePostInput!): Post!
+
+  "Deletes an existing Contact."
+  deleteContact(id: Int!): Contact!
+
+  "Deletes an existing Post."
+  deletePost(id: Int!): Post!
+
+  "Updates an existing Contact."
+  updateContact(id: Int!, input: UpdateContactInput!): Contact!
+
+  "Updates an existing Post."
+  updatePost(id: Int!, input: UpdatePostInput!): Post!
+}
+
+"""
+A blog post.
+"""
+type Post {
+  "The content of a post."
+  body: String!
+
+  "Description for createdAt."
+  createdAt: DateTime!
+
+  "The unique identifier of a post."
+  id: Int!
+
+  "The title of a post."
+  title: String!
+}
+
+"""
+About queries
+"""
+type Query {
+  "Fetch a Contact by id."
+  contact(id: Int!): Contact
+
+  "Fetch Contacts."
+  contacts: [Contact!]!
+
+  "Fetch a Post by id."
+  post(id: Int!): Post
+
+  "Fetch Posts."
+  posts: [Post!]!
+
+  "Fetches the Redwood root schema."
+  redwood: Redwood
+}
+
+"""
+The RedwoodJS Root Schema
+
+Defines details about RedwoodJS such as the current user and version information.
+"""
+type Redwood {
+  "The current user."
+  currentUser: JSON
+
+  "The version of Prisma."
+  prismaVersion: String
+
+  "The version of Redwood."
+  version: String
+}
+
+scalar Time
+
+"""
+Autogenerated input type of UpdateContact.
+"""
+input UpdateContactInput {
+  "The email of the contact."
+  email: String
+
+  "The message of the contact."
+  message: String
+
+  "The name of the contact."
+  name: String
+}
+
+"""
+Autogenerated input type of UpdatePost.
+"""
+input UpdatePostInput {
+  "The content of a post."
+  body: String
+
+  "The title of a post."
+  title: String
+}
+```
+
+#### Root Schema
+
+Documentation is also generated for thr Redwood Root Schema that defines details about Redwood such as the current user and version information.
+```
+type Query {
+  "Fetches the Redwood root schema."
+  redwood: Redwood
+}
+
+"""
+The Redwood Root Schema
+
+Defines details about Redwood such as the current user and version information.
+"""
+type Redwood {
+  "The current user."
+  currentUser: JSON
+
+  "The version of Prisma."
+  prismaVersion: String
+
+  "The version of Redwood."
+  version: String
+}
+
+scalar BigInt
+scalar Date
+scalar DateTime
+scalar JSON
+scalar JSONObject
+scalar Time
+
+```
+
+### Preview in GraphiQL
+
+The [GraphQL Playground aka GraphiQL](https://www.graphql-yoga.com/docs/features/graphiql) is a way for you to interact with your schema and try out queries and mutations. It can show you the schema by inspecting it. You can find the GraphQL Playground at http://localhost:8911/graphql when your dev server is running.
```

Comment:
```suggestion
The [GraphQL Playground aka GraphiQL](https://www.graphql-yoga.com/docs/features/graphiql) is a way for you to interact with your schema and try out queries and mutations. It can show you the schema by inspecting it. You can find the GraphQL Playground at [http://localhost:8911/graphql](http://localhost:8911/graphql) when your dev server is running.
```